FaunaDB.Client/FaunaDB.Client.csproj

```diff
@@ -87,5 +87,7 @@
     <Compile Include="Query\PageHelper.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="Types\BytesV.cs" />
+    <Compile Include="Reflection\Encoder.cs" />
```

Comment:
Not sure if I like `Reflection` namespace. We're not providing a reflection API.
`Codec`? `Encoding`? Idk.